python/helpers/requirements.txt

```diff
@@ -2,6 +2,8 @@ pip==24.0
 pip-tools==7.4.1
 flake8==7.1.0
 hashin==1.0.1
+# TODO: Remove "legacy-cgi" dependency when hashin is updated to support Python 3.13
```

Comment:
```suggestion
# TODO: Remove "legacy-cgi" dependency when we update `hashin` to v1.0.3 which supports Python 3.13.
# But `hashin` v1.0.3 also dropped support for Python 3.8 so first we have to drop 3.8 support in Dependabot.
```

Reply:
True, I'd forgotten about conditional imports.

Reply:
Thanks to https://github.com/dependabot/dependabot-core/pull/11115 getting merged, this conditional dependency is no longer required. I destroyed the commit.